storage/tokudb/ha_tokudb_alter.cc

```diff
@@ -23,25 +23,713 @@ Copyright (c) 2006, 2015, Percona and/or its affiliates. All rights reserved.
 
 #ident "Copyright (c) 2006, 2015, Percona and/or its affiliates. All rights reserved."
 
-#if TOKU_INCLUDE_ALTER_56
-
-#if 100000 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 100099
-#define TOKU_ALTER_RENAME ALTER_RENAME
-#elif (50600 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50699) || \
-      (50700 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50799)
 #define TOKU_ALTER_RENAME ALTER_RENAME
```

Comment:
Remove as well?

Reply:
Oops, yes. Not sure how this slipped through grep foo